### PR TITLE
DOC: fixes broken adonis link

### LINKED ISSUE #310: DOCS: broken link in Adonis docs
**Bug Description**

The "for more information" Link at the end of [this page](https://docs.qiime2.org/2021.4/plugins/available/diversity/adonis/) is broken. Here's the bad link, just in case this is only breaking locally or temporarily:
http://cc.oulu.fi/~jarioksa/softhelp/vegan/html/adonis.html

**Questions**
Do we want to link directly to the r-vegan docs, and tell users to search for adonis?
https://cran.r-project.org/web/packages/vegan/vegan.pdf

### PR DESCRIPTION
**Pull request by ChrisKeefe**
closes #310 

Search didn't turn up any other `oulu.fi` URLs.

### CODE REVIEW

File: q2_diversity/plugin_setup.py [hunk 1]
```diff
@@ -794,7 +794,8 @@
                  'This action differs from beta_group_significance in that it '
                  'accepts R formulae to perform multi-way ADONIS tests; '
                  'beta_group_signficance only performs one-way tests. For '
-                 'more details see http://cc.oulu.fi/~jarioksa/softhelp/vegan/'
-                 'html/adonis.html'),
+                 'more details, consult the reference manual available'
```
Comment (lizgehret) [inline q2_diversity/plugin_setup.py:797]:
```suggestion
                 'more details, consult the reference manual available '
```

**Review by lizgehret — CHANGES_REQUESTED** [review]
Hey @ChrisKeefe! Looks like there is a space missing between a couple of words in your change - other than that, it lgtm!

File: q2_diversity/plugin_setup.py (hunk 1 above)
Comment (ChrisKeefe) [reply q2_diversity/plugin_setup.py]: :facepalm:

**Review by lizgehret — APPROVED** [review]
lgtm!

**Pull request merged**